homeassistant/components/smappee_official/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+  "domain": "smappee_official",
```

Comment:
I realized now that we already have a smappee integration. We probably want to update that integration instead of adding a new integration.

What are the differences between this new integration and the existing integration? They both seem to use the official api by smappee. Is that correct?

Reply:
> We probably want to update that integration
Or replace

Reply:
> What are the differences between this new integration and the existing integration? They both seem to use the official api by smappee. Is that correct?
They indeed use the same official api. The existing integration is outdated and we decided to build a new one from within the company to support newer products and data streams.

Reply:
Good! Since we want to replace the existing integration we should make sure to break as little as possible for users. Eg the existing integration has a sensor platform and a switch platform. That means we need to also add a switch platform in this PR.

@bsmappee please review the existing integration and check what endpoints, ie sensor and switch entity types it exposes. Will the same entities be exposed after this PR?

We basically need to find if there will be any breaking changes and minimize or document them.

Reply:
@MartinHjelmare we use another pypi api wrapper package. The existing integration hasn't been renewed in the last two years. Should be easier to document the elements that haven't changed.

Reply:
We'll replace the existing integration but keep the same name. So we need to track any breaking changes for users.

Reply:
> Eg the existing integration has a sensor platform and a switch platform. That means we need to also add a switch platform in this PR.

I have re-added the switch platform in this PR ([4ee06ed](https://github.com/home-assistant/core/pull/36445/commits/4ee06ed45bbcabef357995d321b590ef57a98241))

Reply:
Yes.